feedback_pipeline.py

```diff
@@ -2263,13 +2271,14 @@ def _generate_view_lists(query):
                 ))
 
                 pkg_ids = query.pkgs_in_view(view_conf_id, arch, output_change="ids")
+                pkg_binary_names = query.pkgs_in_view(view_conf_id, arch, output_change="binary_names")
                 pkg_source_names = query.pkgs_in_view(view_conf_id, arch, output_change="source_names")
 
                 file_name = "view-binary-package-list--{view_conf_id}--{arch}".format(
                     view_conf_id=view_conf_id,
                     arch=arch
                 )
-                _generate_a_flat_list_file(pkg_ids, file_name, query.settings)
```

Comment:
This will be fine in most cases, except multilib. There are cases on x86_64 of one package being present in both x86_64 and i686 arches — having the same name, being counted and listed twice in all other lists except this one that just have the names.

So I wonder if we should, instead of changing the list of NEVRAs into a list of names, add a new list with the names?

You have already set almost everything up for that to work. All you'd need to add is:

```
file_name = "view-binary-package-name-list--{view_conf_id}--{arch}".format(
    view_conf_id=view_conf_id,
    arch=arch
)
_generate_a_flat_list_file(pkg_binary_names, file_name, query.settings)
```

... as a third file to be generated (instead of changing the existing one). And then show it in the UI. That means adding:

```
<li><a href="view-binary-package-name-list--{{view_conf.id}}--{{arch}}.txt">Binary package names</a></li>
```

... in `templates/view_compose_packages.html` around line 66. 

You could also change the other two links so they show:

* Binary package NEVRAs
* Binary package names
* Source package names

What do you think?

Reply:
I think that's a good idea.  I'll do that.

Reply:
And done.